cppcache/integration-test/ThinClientSecurityHelper.hpp

```diff
@@ -59,13 +59,15 @@ const char* regionNamesAuth[] = {"DistRegionAck"};
 std::shared_ptr<CredentialGenerator> credentialGeneratorHandler;
 
 std::string getXmlPath() {
-  char xmlPath[1000] = {'\0'};
-  const char* path = std::getenv("TESTSRC");
-  ASSERT(path != nullptr,
+  std::string path = std::string(std::getenv("TESTSRC"));
```

Comment:
Be careful when assigning the output of std::getenv to an std::string. If the env var is not defined it will return a nullptr, and passing a nullptr to a std::string constructor result in a coredump. There is an utility function Utils::getEnv which acts safely by returning an empty string when the variable is not defined.

Reply:
Oh yes, good catch!!

Reply:
N/A. This function has been removed entirely, along with all the tests that use it, which were already disabled.